Make page_count and current accessible from Twig

kylxbn opened the pull request:
There are times when it is necessary to show the page_count and current page in Twig templates (for example, "2/6" to mean "page 2 of 6"). This pull request aims to expose variables `current`, `items_per_page`, and `page_count`.

I am not good with making pull requests, so if there is any mistake, please tell me, and I will try my best to fix it. Thanks!

kylxbn committed:
Make page_count and current accessible from Twig

There are times when it is necessary to show the page_count and current page in Twig templates (for example, "2/6" to mean "page 2 of 6"). This pull request aims to expose variables `current`, `items_per_page`, and `page_count`.

I am not good with making pull requests, so if there is any mistake, please tell me, and I will try my best to fix it. Thanks!

mahagr reviewed (commented):
Looks good to me, except that it would be nice to have at least docblocks to specify the return type. Though as we're on PHP 7.3+, int return type would be preferable.

kylxbn committed:
Add @return annotations

I had to use `intval()` because `ceil()` returns a `double` by default, so I wanted to cast it to `int` first.

kylxbn commented:
@mahagr , thank you very much for your feedback. I apologize for not adding docblocks in advance, because I wasn't sure which type the variables were in. However, after making sure that all are `int`egers (after casting one `double` to `int` because `ceil()` returns a `double`), I added proper \@return annotations.

I hope this is okay now. If there are any other mistakes, please tell me!

mahagr commented on classes/plugin/PaginationHelper.php:
```diff
@@ -56,7 +56,7 @@ public function __construct(Collection $collection)
 
         $this->items_per_page = $params['limit'];
-        $this->page_count = ceil($collection->count() / $this->items_per_page);
+        $this->page_count = intval(ceil($collection->count() / $this->items_per_page));
```
Casting with `(int)` is the recommended way.

mahagr commented:
Looks good to me!

kylxbn committed:
Use int cast instead of intval()

Sorry!

kylxbn commented:
@mahagr , thank you very much for your patience. I originally intended to use casts but I was unsure, and the internet misguided me. I apologize.

deadda7a commented:
Hey, I would like to heave this feature. I can try to bring the PR up-to-date with the current codebase?

edit: In fact, the code just works like it is now in the PR.
